tests/vision/checks/methodology/simple_feature_contribution_test.py

```diff
@@ -57,18 +58,35 @@ def ret_func(batch):
     return ret_func
 
 
+def test_is_float_column():
+    col = pd.Series([1, 2, 3, 4, 5])
+    assert_that(SimpleFeatureContribution.is_float_column(col), equal_to(False))
+
+    col = pd.Series(['a', 'b', 'c'])
+    assert_that(SimpleFeatureContribution.is_float_column(col), equal_to(False))
+
+    col = pd.Series(['a', 'b', 5.5])
+    assert_that(SimpleFeatureContribution.is_float_column(col), equal_to(False))
+
+    col = pd.Series([1, 2, 3, 4, 5], dtype='float')
+    assert_that(SimpleFeatureContribution.is_float_column(col), equal_to(False))
+
+    col = pd.Series([1, 2, 3, 4, 5.5], dtype='float64')
+    assert_that(SimpleFeatureContribution.is_float_column(col), equal_to(True))
+
+
 def test_no_drift_classification(mnist_dataset_train):
     # Arrange
     train, test = mnist_dataset_train, mnist_dataset_train
-    check = SimpleFeatureContribution(per_class=False)
+    check = SimpleFeatureContribution(per_class=False, random_state=42)
 
     # Act
     result = check.run(train, test)
 
     # Assert
     assert_that(result.value, has_entries({
-        'train': has_entries({'Brightness': equal_to(0)}),
-        'test': has_entries({'Brightness': equal_to(0)}),
+        'train': has_entries({'Brightness': close_to(0.08, 0.005)}),
+        'test': has_entries({'Brightness': close_to(0.08, 0.005)}),
```

Comment:
Any idea why this changed?

Reply:
Previously this was treated as regression. For classification the results are different.